### PR TITLE
Feature/summary usage

### PR DESCRIPTION
**Pull request by MsweliP**

### CODE REVIEW

**Review by laurent-mbuyu — APPROVED** [review]

File: waterlog/src/Components/DailyUsage.js [hunk 1]
```diff
@@ -1,34 +1,54 @@
...
+    labels: labelX,
     datasets: [
       {
-        label: 'Temperature',
-        data: [10,19,27,23,22,4,100,25,23,24,20,19],
+        label: 'Liters',
+        data: dataY,
         fill: true,         
-        borderColor: 'blue'  
+        borderColor: 'red',
```
Comment (CilliersN) [inline waterlog/src/Components/DailyUsage.js:17]: set to rgba

File: waterlog/src/Components/DailyUsage.js [hunk 2]
```diff
@@ -1,34 +1,54 @@
...
+    scales: {
+      xAxes: [ {
+        scaleLabel: {
+          display: true,
+          labelString: 'Hours'
+        },
+        ticks: {
+          major: {
+            fontStyle: 'bold',
+            fontColor: '#FF0000'
```
Comment (CilliersN) [inline waterlog/src/Components/DailyUsage.js:32]: keep convention consistent. line 18 is rgba. check repo convention and stick to one. not 3 (rgba,hex, text['red']
)

File: waterlog/src/Components/SeasonalUsage.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    scales: {
+      xAxes: [ {
+        scaleLabel: {
+          display: true,
+          labelString: 'Seasons'
+        },
+        ticks: {
+          major: {
+            fontStyle: 'bold',
+            fontColor: '#FF0000'
```
Comment (CilliersN) [inline waterlog/src/Components/SeasonalUsage.js:32]: rgba

File: waterlog/src/Components/Usage.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+      
+    }
+  
+    openGraph = (graphType) => {
+      this.setState({
+        display: graphType
+      })
+    }
+
+    getGraphType = ()=>{
```
Comment (CilliersN) [inline waterlog/src/Components/Usage.js:32]: why not switch statement?

File: waterlog/src/Components/Usage.js [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+            <div className="tabcontent">
+              {this.getGraphType()}
+            </div> 
+          </div>
+      )
+    }
+}
+
+Usage.propTypes = {
+	fetchUsageDaily: PropTypes.func.isRequired,
```
Comment (CilliersN) [inline waterlog/src/Components/Usage.js:74]: indentation

File: waterlog/src/reducers/UsageGraphMonthlyReducer.js [hunk 6]
```diff
@@ -0,0 +1,22 @@
+import { FETCH_USAGE_MONTHLY } from '../actions/Types';
+
+const initialState = {
+  item: {
+    dataPoints: [
+        { x: "0", y: "0"}
+    ]
+  },
```
Comment (CilliersN) [inline waterlog/src/reducers/UsageGraphMonthlyReducer.js:8]: put on line 7

File: waterlog/src/reducers/UsageGraphDailyReducer.js [hunk 7]
```diff
@@ -0,0 +1,22 @@
+import { FETCH_USAGE_DAILY } from '../actions/Types';
+
+const initialState = {
+  item: {
+    dataPoints: [
+        { x: "0", y: "0"}
+    ]
+  },
```
Comment (CilliersN) [inline waterlog/src/reducers/UsageGraphDailyReducer.js:8]: put on line 7

File: waterlog/src/reducers/UsageGraphSeasonallyReducer.js [hunk 8]
```diff
@@ -0,0 +1,23 @@
+import { FETCH_USAGE_SEASONALLY } from '../actions/Types';
+
+const initialState = {
+    items:[],
+  item: {
+    dataPoints: [
+        { x: "0", y: "0"}
+    ]
+  },
```
Comment (CilliersN) [inline waterlog/src/reducers/UsageGraphSeasonallyReducer.js:9]: put on line 8

File: waterlog/src/actions/UsageDaily.js [hunk 9]
```diff
@@ -0,0 +1,35 @@
+import { FETCH_USAGE_DAILY, FETCH_USAGE_MONTHLY, FETCH_USAGE_SEASONALLY} from './Types';
```
Comment (CilliersN) [inline waterlog/src/actions/UsageDaily.js:1]: multiline

**Pull request merged**